### PR TITLE
Created POST route to /students/pd_skills endpoint 

### PR DESCRIPTION
**Pull request by sadatakhtar**
Created POST route to /students/pd_skills endpoint (MUST ADD profile key and value for validation within POST method schema)

### CODE REVIEW

**Review by ISTANBULBEKLE — APPROVED** [review]

File: src/routes/students/controllers.js [hunk 1]
```diff
@@ -24,3 +24,18 @@ export const getStudentProfile = async (req, res) => {
     return res.status(400).send("Could not get students");
   }
 };
+
+export const postPdSkills = async (req, res) => {
+  try {
+    let myPdPost = req.body;
+    !myPdPost.profile ? res.send('invalid data schema, need to add profile within POST!') : (pdSkillsPostData.push(myPdPost), res.status(200).json(myPdPost));
```
Comment (Wolfjawan) [inline src/routes/students/controllers.js]: What myPdPost stand for?

Comment (Wolfjawan) [reply src/routes/students/controllers.js]: Also remove empty space please

**Review by Wolfjawan — CHANGES_REQUESTED** [review]

File: src/routes/students/controllers.js (hunk 1 above)
Comment (sadatakhtar) [reply src/routes/students/controllers.js]: myPdPost stands for the pd POST body which is being posted to the end point, renamed to pdPostBody

Comment (sadatakhtar) [reply src/routes/students/controllers.js]: empty space removed

**Review by Wolfjawan — APPROVED** [review]
LGTM 👍

**Review by ISTANBULBEKLE — APPROVED** [review]

**Comment by osagiestar** [comment]
Great job!

**Pull request merged**